index.html

```diff
@@ -2,59 +2,58 @@
 layout: default
 ---
 <div class="home">
-    <div class="posts">
-    {% for post in paginator.posts %}
-        <div class="slice">
-            <div class="container">
-                <h2 class="posts-title">
-                    <a class="no-link-style" href="{{ post.url | prepend: site.baseurl }}">
-                        {{ post.title }}
-                    </a>
-                </h2>
-                <span class="meta">
-                    <time class="posts-date">
-                        <span class="meta-content">
-                            <span class="meta-item">
-                                <i class="far fa-fw fa-calendar-alt"></i> {% include date_translate.html object=post.date %}
-                            </span>
-                            <span class="meta-item">
-                                <i class="far fa-fw fa-clock" title="{% translate global.estimated_reading_time %}"></i> {% include reading_time.html content=post.content %}
-                            </span>
-                        </span>
-                    </time>
-                    {% include author_link.html authors=post.authors %}
-                </span>
-                <p class="excerpt">
-                    {{ post.excerpt }}
-                </p>
+    <div class="article-container">
+        {% for post in paginator.posts %}
+        <div class="article">
```

Comment:
To avoid confusion, maybe use a better class naming ? "article_excerpt"

Reply:
changé pour => article-preview